content/installation/user/kerberos.Rmd

```diff
@@ -160,15 +159,6 @@ Description=Microsoft ODBC Driver 13 for SQL Server
 Driver=/opt/microsoft/msodbcsql/lib64/libmsodbcsql-13.1.so.4.0
 UsageCount=2
 ```
-<<<<<<< HEAD:content/installation/user/kerberos.md
-
-To get a list of all databases on a server (make sure you have an active ticket):
-
-```
-sqlcmd -S inbo-sql07-prd.inbo.be -E
-1> Select name from sys.databases;
-2> Go
-```
```

Comment:
@ThierryO  The second `sqlcmd` session contained code to show all databases instead of the first 10 only, so it is not a duplicate. But you can choose to keep only one version.